routes/dependencies/remove/dependency.js

```diff
@@ -1,10 +1,14 @@
 const { send } = require('micro');
+const hasYarn = require('has-yarn');
 const ansiCommandStream = require('../../../src/ansiCommandStream');
 
 module.exports = async (req, res) => {
   const { params: { dependency } } = req;
   try {
-    const { stream, kill } = ansiCommandStream({ args: ['rm', dependency] });
+    const removeCommand = hasYarn() ? 'remove' : 'uninstall';
```

Comment:
I actually meant the mapping could be done in ansiCommand but this works too I guess?

Reply:
Seems fine to be here.

Reply:
Yeah, it's the same, if you want i can change it

Reply:
It’s fine, I was thinking of a different architecture where you just send the name of the command (run, remove, add, addDev) to that function and it would figure out what to actually use. But this works too